File: 백준/Bronze/5585. 거스름돈/거스름돈.py
# 5585 : 거스름돈
import sys
input = sys.stdin.readline

n = 1000 - int(input().rstrip())
arr = [500, 100, 50, 10, 5, 1]
cnt = 0
for a in arr:
    cnt += n // a
    n %= a
print(cnt)

File: 백준/Bronze/5585. 거스름돈/거스름돈.cc
#include <stdio.h>

int main(void) {
    int n;
    scanf("%d", &n);
    n = 1000 - n;

    int cnt = 0;
    cnt += n / 500;
    n %= 500;
    cnt += n / 100;
    n %= 100;
    cnt += n / 50;
    n %= 50;
    cnt += n / 10;
    n %= 10;
    cnt += n / 5;
    n %= 5;
    cnt += n;

    printf("%d", cnt);

    return 0;
}